atc/policy/checker.go

```diff
@@ -0,0 +1,144 @@
+package policy
+
+import (
+	"fmt"
+	"strings"
+
+	"code.cloudfoundry.org/lager"
+	"github.com/jessevdk/go-flags"
+)
+
+const ActionUsingImage = "UsingImage"
```

Comment:
Probably best to match the imperative tone of the API handlers:

```suggestion
const ActionUseImage = "UseImage"
```

Reply:
Renamed.